internal/upload/koji/rh-logrus-adapter.go

```diff
@@ -0,0 +1,34 @@
+package koji
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type LeveledLogrus struct {
+	*logrus.Logger
+}
+
+func (l *LeveledLogrus) fields(keysAndValues ...interface{}) map[string]interface{} {
```

Comment:
I think I would probably make this function a non-member one as it doesn't need access to anything from `l *LeveledLogrus`.

Reply:
Follow-up?